### PR TITLE
Eusebiusyw beijing

### LINKED ISSUE #15: ZHAOZHENQIAN tracks progress on GH Flow
- [x] Create a branch
- [x] Add File/Creating Commits
- [x] Open a PR
- [x] Have a discussion about the PR
- [ ] Add commits
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by Eusebiusyw**
@hectorsector 
1. What's the difference between a branch and a fork?

closes #15

### CODE REVIEW

File: Eusebiusyw-Beijing.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Beijing, China
+
+#### Places to Eat
+Test
```
Comment (tdrobbin) [inline Eusebiusyw-Beijing.md:4]: I think that's a bad idea

File: Eusebiusyw-Beijing.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## Beijing, China
```
Comment (zhaozhenqian) [inline Eusebiusyw-Beijing.md:1]: You are also from Beijing?

File: Eusebiusyw-Beijing.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+#### Places to Eat
+Test
+#### Places to Go
+Changes
```
Comment (tdrobbin) [inline Eusebiusyw-Beijing.md:7]: jsj

File: Eusebiusyw-Beijing.md (hunk 2 above)
Comment (Eusebiusyw) [inline Eusebiusyw-Beijing.md:1]: Yea which part are you from?

Comment (zhaozhenqian) [inline Eusebiusyw-Beijing.md:1]: close to the Olympic Park

Comment (Eusebiusyw) [inline Eusebiusyw-Beijing.md:1]: 15 mins from where I live 和平西桥

**Pull request merged**